README.md

```diff
@@ -1,46 +1,74 @@
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+# PushPull (Client-Side/Front-End)
+A *fitness programming* web-app created for rapid prototyping of workout-related programs and serve as a fitness community.
+
+- This repository houses the client-side of my web application *PushPull*. As such, most of the details mentioned here are largely front-end related. If you want to learn about my API (server-side) please visit [PushPull's server-side repository](https://github.com/Haz-git/PushPull-server).
+
+### About The Project
+---
+*PushPull* is my newest passion project designed for the rapid creation of fitness programs for distribution. Whether fitness trainer, professional bodybuilder, or just a general fitness enthusiast like me-- I wanted PushPull to serve as a centralized community where people can create, search, rate, and discuss a variety of workout programs.
+
+During my fitness journey, I jumped from one workout program to another when the time felt right. I perused Reddit for hours to find the best workout programs for my current level. Then, after long contemplation, I visited multiple sketchy blog websites to find and download that fitness program. My dislike toward this process led to the creation of *PushPull*
+
+#### Built With
+- TypeScript
+- JavaScript
+- HTML5
+- CSS3
+- ReactJS
+- Redux, React-Redux
+- Styled-Components
+- Mantine (Component Library)
+
+*For more, see **acknowledgements***.
+
+### Usage
+---
+
+1. Head over to [PushPull's Website](https://www.gopushpull.com/).
+2. Navigate to 'Sign up'--create a free account.
+3. Navigate to 'Log In' and enter the app using your credentials.
+4. Start exploring!
+
+### Feature Highlights
+---
+**Here are just a few features in *PushPull***.
+
+The **Landing Page** was designed in collaboration with a UI/UX designer. Authentication system complete with protected routes was developed by integrating the auth provider *UserFront*. The user can sign up or log in at any unprotected page.
+
+![](https://media.giphy.com/media/Ee4JJT2M9Ad3s8BtBV/giphy.gif)
+    
+### Contributing
+---
+Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions or suggestions to this project are appreciated.
+
+1. Fork the Project.
+2. Create your Feature Branch (git checkout -b feature/yourFeature).
+3. Commit your Changes (git commit -m 'Add some yourFeature').
+4. Push to the Branch (git push origin feature/yourFeature).
+5. Open a Pull Request.
+
+### License
+---
+Distributed under the GPL-3.0 License. See ```LICENSE``` for more information.
```

Comment:
you can link to the actual file. also, why GPL?

Reply:
Hmm, I chose this a long time ago. I think my reasoning was that if I wanted to make this a SaaS in the future I didn't want someone to monetize it--does MIT do the same thing?

Reply:
No, thats the correct answer.  MIT definitely does not do that.